week4/README

```diff
@@ -0,0 +1,64 @@
+for deploy kubflow
```

Comment:
please read this: https://github.com/hackergrrl/art-of-readme

this formatting is really hard to read